src/main/kotlin/no/skatteetaten/aurora/databasehotel/dao/DatabaseHotelDataDao.kt

```diff
@@ -11,20 +13,26 @@ interface DatabaseHotelDataDao {
 
     fun createSchemaData(name: String, schemaType: String): SchemaData
 
-    fun findSchemaDataById(id: String): SchemaData?
+    fun findSchemaDataById(id: String, active: Int = 1): SchemaData?
```

Comment:
Kan denne være boolean? Regner med den er 0/1 pga databasen, eller?

Reply:
Ja, men det trenger for så vidt ikke lekke ut i apiet. Godt poeng. Fikser.